core-processor/src/ext.rs

```diff
@@ -603,3 +605,40 @@ impl EnvExt for Ext {
         &self.forbidden_funcs
     }
 }
+
```

Comment:
I'm really versus any default constructors for ext, due to the importance and uniqueness of the inner components as gas allowance counter etc.

This codebase will reduce in future, once many of this params got united into some general structs

Reply:
A good practice is to implement/derive three basic traits for public structs (I call it DDC-rule): `Default`, `Debug`, and `Clone` (see "Rust for Rustaceans" book). But if you promise to refactor this part I am ready to turn a blind eye to that 😄